pkg/user/pruning_test.go

```diff
@@ -45,6 +45,6 @@ func TestPruningInTxTracker(t *testing.T) {
 	txClient.pruneTxTracker()
 	// Prunes the transactions that are 10 minutes old
 	// 5 transactions will be pruned
-	require.Equal(t, txsToBePruned, txTrackerBeforePruning-txsToBePruned)
+	require.Equal(t, txsNotReadyToBePruned, txTrackerBeforePruning-txsToBePruned)
```

Comment:
hum, why wasn't this failing before? if they're equal, we should update the test to have different numbers to make sure we're not missing anything